### DISCUSSION TITLE
Release 0.4 Schedule

### DISCUSSION THREAD

**Discussion by shilpa-padgaonkar** [camaraproject/Commonalities#125] [General]
There was an ask in TSC to check if we need a release 0.4 in Commonalities before a Camara wide release (this is expected to be around May/June timeframe and is being planned in the release management working group).

I see 2 important issues/PRs which will impact this decision: 

- https://github.com/camaraproject/Commonalities/issues/31
- https://github.com/camaraproject/Commonalities/pull/120 (since we are awaiting feedback from subprojects here, there is a chance that this PR might not make the release 0.3)

If we plan to go ahead with PRs for the above 2 points, IMHO we would need to do it before the Camara wide release as this has impacts on several subprojects.

Would be great to have more feedback on this discussion. Also, feel free to point to any additional open issues that need to be closed before the planned Camara wide release but are not part of the planned 0.3 release.

**Comment by bigludo7** [comment]
Hi @shilpa-padgaonkar 

Yes make completely sense to have this discussion.

For commonalities release 4.0 
•	for #31 I saw a debate on the issue and we have not an alignement so far . Probably we should ask a ‘formal’ positioning from our community member?
•	For [Update CAMARA_common.yaml: add Device headers, fix address by Kevsy · Pull Request #120 · camaraproject/Commonalities (github.com)](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcamaraproject%2FCommonalities%2Fpull%2F120&data=05%7C02%7Cludovic.robert%40orange.com%7C96350809d0de4136df3c08dc1e846143%7C90c7a20af34b40bfbc48b9253b6f5d20%7C0%7C0%7C638418801123155550%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=OL%2F7dvUdanPTJKm0doUDIhOkF1mxvZZfITwW7%2Fqe3w4%3D&reserved=0) we support this. I did not see real blocking points for this one.
•	A related point – for us it was very important to close the Test Definition topic and very happy that we have merged it. Thanks a lot for your effort. It is probably important to have all these test definition for the wide release.


One other think where I would like to get community view… As live CAMARA implementation arise around the world we should trigger some “feedback loop” with the companies consuming our API to get their pain points. It could perhaps trigger some request for update.
